index.js

```diff
@@ -342,7 +342,8 @@ class K8sExecutor extends Executor {
 
                 if (resp.body.spec && resp.body.spec.nodeName) {
                     updateConfig.stats = {
-                        hostname: resp.body.spec.nodeName
+                        hostname: resp.body.spec.nodeName,
+                        imagePullStartTime: (new Date()).toISOString()
```

Comment:
is this precise ?

Reply:
what do you mean by precise? this is not the exact same when it starts pulling the image, but should be a good estimate. it's in the same format as enqueue time